homeassistant/components/workday/binary_sensor.py

```diff
@@ -77,7 +82,9 @@
     "Northern Ireland",
     "Norway",
     "NO",
-    "Polish",
```

Comment:
`Polish` is still valid but deprecated. This change would break existing setups. Thus this change would make the PR a "Breaking change".

Reply:
Sorry did not know that, wil add it back :)